src/util.ts

```diff
@@ -34,6 +34,16 @@ export async function sendEventLogMessage(guild: Guild, originId: string | null,
 	return logChannel.send({ content, embeds: [embed] });
 }
 
+export async function sendModLogMessage(guild: Guild, embed: EmbedBuilder, content?: string): Promise<Message | null> {
```

Comment:
Why doesn't this use the `channels.event-logs.blacklist` ?

Reply:
I wrote that before I decided to put deleted messages in the mod log, and I forgot to add the blacklist back.